src/main/java/uk/gov/hmcts/reform/civil/model/FlightDelay.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.civil.model;
+
```

Comment:
I would suggest chaing the name of model to FlightDelayDetail?

Reply:
Yes, agree.